data/json/mapgen/house09.json

```diff
@@ -83,36 +83,34 @@
-                { "item" : "dining",               "chance" : 70, "x" : [20,21], "y" : [ 1, 1] },
-                { "item" : "bedroom",              "chance" : 60, "x" : [ 1,10], "y" : [17,21] },
-                { "item" : "dresser",              "chance" : 50, "x" : [ 2, 2], "y" : [17,17] },
-                { "item" : "dresser",              "chance" : 50, "x" : [ 2, 2], "y" : [20,20] },
-                { "item" : "consumer_electronics", "chance" : 20, "x" : [ 1,10], "y" : [17,21] },
-                { "item" : "homeguns",             "chance" : 20, "x" : [ 1,10], "y" : [17,21] },
-                { "item" : "novels",               "chance" : 30, "x" : [ 2, 2], "y" : [ 8,10] },
-                { "item" : "softdrugs",            "chance" : 50, "x" : [15,15], "y" : [15,15] },
-                { "item" : "kitchen",              "chance" : 70, "x" : [ 2,10], "y" : [13,15] },
-                { "item" : "fridge",               "chance" : 70, "x" : [ 4, 4], "y" : [13,13] },
-                { "item" : "oven",                 "chance" : 70, "x" : [ 5, 5], "y" : [13,13] },
-                { "item" : "cannedfood",           "chance" : 20, "x" : [ 9,10], "y" : [15,15] },
-                { "item" : "pasta",                "chance" : 50, "x" : [ 3, 5], "y" : [15,15] },
-                { "item" : "knifeblock",           "chance" : 30, "x" : [ 3, 5], "y" : [15,15] },
-                { "item" : "camping",              "chance" : 10, "x" : [21,21], "y" : [17,18] },
-                { "item" : "dresser",              "chance" : 50, "x" : [20,21], "y" : [21,21] },
-                { "item" : "cleaning",             "chance" : 50, "x" : [15,18], "y" : [21,21] }
+            "place_loot" : [
+                { "group" : "bed",                  "chance" : 90, "x" : [ 2, 3], "y" : [18,19], "repeat" : [1,3] },
+                { "group" : "livingroom",           "chance" : 90, "x" : [12,21], "y" : [ 2, 6], "repeat" : [1,4] },
+                { "group" : "livingroom",           "chance" : 90, "x" : [ 2,10], "y" : [ 6,11], "repeat" : [1,4] },
+                { "group" : "dining",               "chance" : 85, "x" : [16,18], "y" : [ 8, 9], "repeat" : [1,3] },
+                { "group" : "cannedfood",           "chance" : 80, "x" : [20,21], "y" : [11,11], "repeat" : [1,2] },
+                { "group" : "dining",               "chance" : 90, "x" : [20,21], "y" : [11,11], "repeat" : [1,2] },
+                { "group" : "bedroom",              "chance" : 80, "x" : [ 1,10], "y" : [17,21], "repeat" : [1,5] },
+                { "group" : "dresser",              "chance" : 80, "x" : [ 2, 2], "y" : [17,17], "repeat" : [1,3] },
+                { "group" : "dresser",              "chance" : 80, "x" : [ 2, 2], "y" : [20,20], "repeat" : [1,3] },
+                { "group" : "consumer_electronics", "chance" : 60, "x" : [ 1,10], "y" : [17,21], "repeat" : [1,3] },
+                { "group" : "homeguns",             "chance" : 10, "x" : [ 1,10], "y" : [17,21] },
```

Comment:
Again `guns_common` or `guns_pistol_common` (typically the latter should be preferred)